internal/rpc/models.go

```diff
@@ -60,22 +57,22 @@ def regular(fieldName, storedAs, datatype):
     return {
         "type" : type, 
         "title" : fieldName, 
-        "datatype" : datatype
+        "datatype" : datatype.name
     }
 
-def enum(fieldName, id, datatype):
-    evs = FindMany("enumValue", EqFK("datatype", id))
+def enum(fieldName, datatype):
+    dt = aft.findOne("datatype", {"where" : {"id" : datatype.id}, "include" : {"enumValues" : True}})
```

Comment:
you can just findOne("enum")

Reply:
I wasn't sure how to do that referencing the datatype id

Reply:
It's the same ID!

Reply:
Enum implements datatype, so it gets indexed under <enum>/<id> and <datatype>/<id>